src/taxonomy/TaxonomyLayout.scss

```diff
@@ -0,0 +1,3 @@
+.taxonomy-layout {
+  background-color: #E9E6E4;
```

Comment:
This is equivalent to bg-light-400, so you can avoid the custom css.

Reply:
Thank you! Fixed here: 5fba4627c1dbfd05c00aa9394f02307699d67592